packages/desktop-client/src/components/sidebar.js

```diff
@@ -446,13 +447,14 @@ function Accounts({
 function ToggleButton({ style, isFloating, onFloat }) {
   return (
     <View className="float" style={[style, { flexShrink: 0 }]}>
-      <Button type="bare" onClick={onFloat} color={colors.n5}>
+      <Button type="bare" onClick={onFloat}>
         {isFloating ? (
           <Pin
             style={{
               margin: -2,
               width: 15,
               height: 15,
+              color: theme.altpageTextSubdued,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏color of the collapse/expand button is wrong

<img width="196" alt="Screenshot 2023-08-12 at 09 44 28" src="https://github.com/actualbudget/actual/assets/886567/386d2a8c-1bdb-4d32-999b-51400dc2f683">